spec/_attachments/ic.did

```diff
@@ -1,12 +1,18 @@
 type canister_id = principal;
 type wasm_module = blob;
 
+type log_visibility = variant {
```

Comment:
How about the case where the canister would store the logs on the IC? In that case, it might make sense to have the logs accessible by a separate canister that only indexes them, i.e., adding a third option, a (bounded) list of principals who can access the logs.

I guess the most flexible option would be to expose reading the logs as a system API instruction instead of a management call, and then have the canister handle its own access control.

Reply:
>  a (bounded) list of principals who can access the logs.

that used to be part of the initial design, but then it was dropped to keep it simple

Reply:
Ack. I'll resolve this conversation since there's no action points.